docs/BR.md

```diff
@@ -1840,9 +1847,11 @@ A Certificate issued to a Subordinate CA that is an affiliate of the Issuing CA:
 A Subordinate CA SHALL represent, in its Certificate Policy and/or Certification Practice Statement, that all Certificates containing a policy identifier indicating compliance with these Requirements are issued and managed in accordance with these Requirements.
 
 #### 7.1.6.4 Subscriber Certificates
-A Certificate issued to a Subscriber MUST contain a certificatePolicies extension.
+A Subscriber Certificate that is a Server Certificate MUST contain a certificatePolicies extension.
+
+The extension MUST contain one or more policy identifiers that indicate adherence to and compliance with these Requirements. CAs MUST use either a CA/Browser Forum identifier reserved for this purpose or a policy identifier documented by the CA in its Certificate Policy and/or Certification Practice Statement to indicate the Certificate's compliance with these Requirements. 
 
-The extension MUST contain one or more policy identifiers that indicate adherence to and compliance with these Requirements. CAs MUST either use a CA/Browser Forum identifier reserved for this purpose or MUST use a policy identifier documented by the CA in its Certificate Policy and/or Certification Practice Statement to indicate the Certificate's compliance with these Requirements.
+The CA MUST NOT include more than one CA/Browser Forum identifiers reserved for this purpose. The CA MAY include multiple policy identifiers documented by the CA, provided they are not CA/Browser Forum identifiers reserved for this purpose.
```

Comment:
I realize this may look like a new normative requirement, but it was the logical consequence of the existing "must have subject data " and "must not have subject data"